python/pdstools/adm/Reports.py

```diff
@@ -345,3 +352,56 @@ def _find_quarto_executable(self):
         raise FileNotFoundError(
             "Quarto executable not found. Please ensure Quarto is installed and in the system PATH."
         )
+
+    def excel(self, file: Path = Path("Tables.xlsx"), predictorBinning=False):
```

Comment:
Should we just call this excel or maybe `generate.excel_export()`?

Reply:
Renamed it. Should we merge it to keep things in sync?